mods/default/init.lua

```diff
@@ -29,11 +29,10 @@ mod = LuaMod.new("default")
 dofile("mods/default/blocks.lua")
 dofile("mods/default/items.lua")
 dofile("mods/default/recipes.lua")
+dofile("mods/default/biomes.lua")
 
 openminer:world():terrain_generator():set_blocks({
```

Comment:
This shoudn't be needed anymore, all the blocks should be defined per-biome only